packages/relay/src/services/utils/ProofHelper.ts

```diff
@@ -50,7 +50,8 @@ class ProofHelper {
     async getAndVerifyEpochKeyLiteProof(
         publicSignals: PublicSignals,
         proof: Groth16Proof,
-        synchronizer: UnirepSocialSynchronizer
+        synchronizer: UnirepSocialSynchronizer,
+        onCancel: boolean = false
```

Comment:
I suggest to rename `onCancel` to `enableEpochValidation` because main purpose of this function is to verify a proof, we should follow the scope of the purpose to name the parameters. `onCancel` is too focused on voting feature, but this function is not only used on voting.

Reply:
modified